public/user-view-movie.php

```diff
@@ -0,0 +1,74 @@
+<?php
+include '../src/database/credentials.php';
+?>
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <title>And Action</title>
+  <?php include "../templates/head.php"?>
+  <link rel="stylesheet" href="./assets/css/director-movie-specific.css">
+  <link rel="stylesheet" href="./assets/css/styleupload.css">
+  <link rel="stylesheet" href="./assets/css/thumpnail-display.css">
+  <link rel="stylesheet" href="./assets/css/userview.css">
+</head>
+
+<body>
+  <!-- start navbar -->
+
+  <div class="navbar">
+    <?php include "../templates/navbar.php";?>
+  </div>
+
+  <!-- end navbar -->
+  <!-- start main container -->
+
+  <div class="container container-movie-specific">
+    <div class="text-wrapper">
+        <div class="movie">
+            <iframe src="" title="placeholder" height="300" width="600"></iframe>
```

Comment:
iframe? laat dit maar zo staan, alleen gebruiken we het video element, niet de iframe.